engine/configurator/conf_main.c

```diff
@@ -2948,13 +2948,8 @@ main(int argc, char **argv)
         goto exit;
     }
 
-    if ((filename = (char *)malloc(strlen(tmp_dir) +
-                                   strlen("/te_cfg_tmp.yml") + 1)) == NULL)
-    {
-        ERROR("No enough memory");
-        goto exit;
-    }
-    sprintf(filename, "%s/te_cfg_tmp.yml", tmp_dir);
+    te_string_append(&filename_temp, "%s/te_cfg_tmp.yml", tmp_dir);
+    filename = filename_temp.ptr;
```

Comment:
I've actually meant using `te_file_create_unique()`. Also there is `te_string_fmt()` which makes intermediate `te_string` unnecessary.

Reply:
It seems to me that `te_file_create_unique()` is not suitable here. Thanks, I've used `te_string_fmt()`